### PR TITLE
Fixup the layer importer

### PR DESCRIPTION
**Pull request by weskamm**
This merges the changes made to the GeoServer Importer.
Related to https://github.com/terrestris/momo3-backend/pull/123

### CODE REVIEW

File: app/view/panel/layer/GeneralController.js [hunk 1]
```diff
@@ -271,12 +281,16 @@ Ext.define('MoMo.admin.view.panel.layer.GeneralController', {
         //Load newly created Layerdata
         viewModel.set('entityId', respObj.data.id);
         coeLayerController.loadLayerData();
-        // reset data and gui
+        // reset data and guit
```
Comment (hwbllmnn) [inline app/view/panel/layer/GeneralController.js:284]: typo

**Comment by hwbllmnn** [comment]
Looks good, but I'm unsure why travis fails currently. Maybe we should fix that before merging.

**Comment by weskamm** [comment]
Thanks for review, it fails because of ol3 master being unable to build. Will get fixed in https://github.com/terrestris/momo3-admin/pull/72

**Comment by hwbllmnn** [comment]
Ok, 👍 , please merge!

**Comment by weskamm** [comment]
travis green again, merging now

**Pull request merged**